src/math/polynomial.lisp

```diff
@@ -0,0 +1,104 @@
+(in-package :lbge.math)
+
+(defclass polynomial ()
+  ((coefficients
```

Comment:
I think `:documentation` with an example would be nice.
Here polynomials are written from the 0-th power to the highest, while in math its usually vice versa.